<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Form</class>
 <widget class="QWidget" name="Form">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>500</width>
    <height>600</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>0</width>
    <height>0</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>16777215</width>
    <height>16777215</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>SwitchControl</string>
  </property>
  <property name="styleSheet">
   <string notr="true">QWidget {
	background-color: rgb(58, 58, 58);
}
QScrollArea {
	border: none;
}
QScrollBar:vertical {
	width: 10px;
	background: rgba(0, 0, 0, 0);
	border: none;
	margin: 0 0 0 0;
}
QScrollBar::sub-line:vertical, QScrollBar::add-line:vertical, QScrollBar::sub-page:vertical, QScrollBar::add-page:vetical {
	background: none;
	border: none
}
QScrollBar::handle:vertical {
	background-color: rgb(110, 133, 211);
	border: none;
}
QScrollBar::handle:vertical:hover {
	background-color: rgb(125, 151, 239);
}
QScrollBar::handle:vertical:pressed {
	background-color: rgb(88, 106, 234);
	background-color: rgb(81, 110, 203);
}


QScrollBar:horizontal {
	height: 10px;
	background: rgba(0, 0, 0, 0);
	border: none;
	margin: 0 0 0 0;
}
QScrollBar::sub-line:horizontal, QScrollBar::add-line:horizontal, QScrollBar::sub-page:horizontal, QScrollBar::add-page:vetical {
	background: none;
	border: none
}
QScrollBar::handle:horizontal {
	background-color: rgb(110, 133, 211);
	border: none;
}
QScrollBar::handle:horizontal:hover {
	background-color: rgb(125, 151, 239);
}
QScrollBar::handle:horizontal:pressed {
	background-color: rgb(88, 106, 234);
	background-color: rgb(81, 110, 203);
}</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QScrollArea" name="scrollArea">
     <property name="minimumSize">
      <size>
       <width>0</width>
       <height>0</height>
      </size>
     </property>
     <property name="styleSheet">
      <string notr="true"/>
     </property>
     <property name="widgetResizable">
      <bool>true</bool>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
     <widget class="QWidget" name="scrollAreaWidgetContents">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>0</y>
        <width>478</width>
        <height>578</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true"/>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout_2">
       <property name="spacing">
        <number>0</number>
       </property>
       <property name="leftMargin">
        <number>0</number>
       </property>
       <property name="topMargin">
        <number>0</number>
       </property>
       <property name="rightMargin">
        <number>0</number>
       </property>
       <property name="bottomMargin">
        <number>0</number>
       </property>
       <item>
        <widget class="QWidget" name="container_widget" native="true">
         <property name="minimumSize">
          <size>
           <width>0</width>
           <height>0</height>
          </size>
         </property>
         <property name="styleSheet">
          <string notr="true">QWidget {
	color: rgb(255, 255, 255);
}
QPushButton {
	color: rgb(255, 255, 255);
	background-color: rgb(0, 116, 228);
	border: none;
	border-radius: 6px;
}
QPushButton:hover {
	background-color: rgb(40, 138, 232);
}
QPushButton:pressed {
	background-color: rgb(61, 149, 234);
}</string>
         </property>
         <layout class="QVBoxLayout" name="verticalLayout_3">
          <item>
           <widget class="QGroupBox" name="classic_group_box">
            <property name="font">
             <font>
              <family>Microsoft YaHei</family>
             </font>
            </property>
            <property name="title">
             <string>Classic</string>
            </property>
            <layout class="QVBoxLayout" name="verticalLayout_4">
             <item>
              <widget class="QLabel" name="label">
               <property name="font">
                <font>
                 <family>Microsoft YaHei</family>
                 <pointsize>9</pointsize>
                </font>
               </property>
               <property name="text">
                <string>SwitchControl with Classic style</string>
               </property>
               <property name="alignment">
                <set>Qt::AlignCenter</set>
               </property>
              </widget>
             </item>
            </layout>
           </widget>
          </item>
          <item>
           <widget class="QGroupBox" name="modern_group_box">
            <property name="font">
             <font>
              <family>Microsoft YaHei</family>
             </font>
            </property>
            <property name="title">
             <string>Modern</string>
            </property>
            <layout class="QVBoxLayout" name="verticalLayout_5">
             <item>
              <widget class="QLabel" name="label_2">
               <property name="font">
                <font>
                 <family>Microsoft YaHei</family>
                 <pointsize>9</pointsize>
                </font>
               </property>
               <property name="text">
                <string>SwitchControl with Modern style</string>
               </property>
               <property name="alignment">
                <set>Qt::AlignCenter</set>
               </property>
              </widget>
             </item>
            </layout>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
